src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -6689,39 +6490,14 @@ namespace HVACVariableRefrigerantFlow {
 		// PURPOSE OF THIS SUBROUTINE:
 		// This subroutine updates the node data for the VRF Condenser.
 
-		// METHODOLOGY EMPLOYED:
-		// na
-
-		// REFERENCES:
-		// na
-
-		// USE STATEMENTS:
-		// na
-
-		// Locals
-		// SUBROUTINE ARGUMENT DEFINITIONS:
-
-		// SUBROUTINE PARAMETER DEFINITIONS:
-		// na
-
-		// INTERFACE BLOCK SPECIFICATIONS
-		// na
-
-		// DERIVED TYPE DEFINITIONS
-		// na
-
 		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-		//    INTEGER :: CondenserInletNode  !- inlet node for VRF water-cooled condenser
 		int CondenserOutletNode; // - outlet node for VRF water-cooled condenser
 
-		//    CondenserInletNode = VRF(VRFCond)%CondenserNodeNum
 		CondenserOutletNode = VRF( VRFCond ).CondenserOutletNodeNum;
 
 		Node( CondenserOutletNode ).Temp = VRF( VRFCond ).CondenserSideOutletTemp;
 
-		//    Node(CondenserInletNode)%MassFlowRate = CondenserWaterMassFlowRate
 		Node( CondenserOutletNode ).MassFlowRate = CondenserWaterMassFlowRate;
-
 		Node( CondenserOutletNode ).MassFlowRateMaxAvail = Node( CondenserOutletNode ).MassFlowRateMaxAvail;
 		Node( CondenserOutletNode ).MassFlowRateMinAvail = Node( CondenserOutletNode ).MassFlowRateMinAvail;
```

Comment:
Well these 2 lines look suspect. Should be outlet = inlet.

Reply:
Well, that uneasy feeling of not knowing exactly how these Min/Max/MinAvail/MaxAvail variables is upon me.

1) UpdateVRFCondenser is only called for water-cooled equipment
2) Water based equipment should not set Min/MaxAvail since SetComponentFlowRate handles that. These lines don't hurt anything here since it writes outlet = outlet (but should be removed eventually).
3) Setting Node( CondenserOutletNode ).MassFlowRate = CondenserWaterMassFlowRate; here is OK but redundant since SetComponentMassFlowRate already did that. 
4) I noticed the VRF TU's are not passing MinAvail and MaxAvail through to the coil outlet node as do other air-side equipment. Probably works because TU's are zone equipment. 
5) Do no harm, no changes here at this point.